lib/network/rest/utils.js

```diff
@@ -0,0 +1,13 @@
+'use strict'; // eslint-disable-line
+
+module.exports.explodePath = function explodePath(path) {
+    const pathMatch = /^(\/[a-zA-Z0-9]+)(\/([0-9a-f]*))?$/.exec(path);
+    if (pathMatch) {
+        return {
+            service: pathMatch[1],
+            key: (pathMatch[3] !== undefined && pathMatch[3].length > 0 ?
+                  pathMatch[3] : undefined),
+        };
+    }
+    return undefined;
```

Comment:
So in order to know if an error occurs, do we have to check the type of what's returned? (`if (typeof ret === 'Error') {...}` for example) I personally find the `try..catch` a better structure (it is actually made for that and can possibly be placed further up in the stack if required). But if you have a better way to handle returned errors, I'd be glad to know! (real question, no sarcasm)